### ISSUE TITLE
We need better comments

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/liquidpledging#54]
they should follow natspec and be consistant with the comments in the LiquidPledgingBase.

**Comment by Quazia** [comment]
Is the main natspec issue the whole @dev thing? This just means updating LiquidPledging to follow the changes in LiquidPledgingBase?

**Comment by GriffGreen** [comment]
Yes it's the natspec thing, I really need to make a commenting guidelines wiki post...
